src/app/common/modal/container-modal/container-modal.component.html

```diff
@@ -0,0 +1,24 @@
+<div role="dialog" cdkTrapFocus [cdkTrapFocusAutoCapture]="isCdkFocusInitial">
```

Comment:
Do we need `role="dialog"` here?  If you inspect the DOM, it is automatically added to the `<modal-container>` that ngx-bootstrap creates.

Reply:
Confirmed that we don't and removed this `role="dialog"`